lib/internal/bootstrap/realm.js

```diff
@@ -144,6 +143,9 @@ const experimentalModuleList = new SafeSet();
 {
   const bindingObj = { __proto__: null };
 
+  // TODO(@jasnell): If the --deprecated-process-binding option is false we really
```

Comment:
You can just expose the `processBindingAllowList` list, which is set to empty initially, and add things back in pre-execution. We do something similar for `--expose-internals` for example.